src/main/java/walle/WallE.java

```diff
@@ -0,0 +1,220 @@
+package walle;
+
+import java.lang.reflect.Array;
+import java.util.ArrayList;
+import java.util.Scanner;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+public class WallE {
+    private static final int MAX_LIST_SIZE = 100;
+    private static final String exitMessage = "Bye. Hope to see you again soon!";
+    private static final String greeting = "Hello! I'm Wall-E!\n" + "\tWhat can I do for you?\n\n";
+    private static int listSize = 0;
+    private static final String FILE_NAME = "data.txt";
+    private static ArrayList<Task> tasks = new ArrayList<>();
+
+    public static void printLineBreak() {
+        String lineBreak = "\t_________________________________\n";
+        System.out.print(lineBreak);
+    }
+
+    public static void printWithLineBreak(String response) {
+        printLineBreak();
+        System.out.println("\t" + response);
+        printLineBreak();
+    }
+
+    public static void printWithoutLineBreak(String response) {
+        System.out.println("\t" + response);
+    }
+
+    public static void printGreeting() {
+        printWithLineBreak(greeting);
+    }
+
+    public static void printExitMessage() {
+        printWithLineBreak(exitMessage);
+    }
+
+    public static void printTaskList(int size) {
+        printLineBreak();
+        printWithoutLineBreak("Here are the tasks in your list: ");
+        for (int i = 0; i < size; i++) {
+            printWithoutLineBreak(Integer.toString(i + 1) + ". " + tasks.get(i).toString());
+        }
+        printLineBreak();
+    }
+
+    public static void main(String[] args) throws IOException {
+        printGreeting();
+        FileParser parser = new FileParser("data.txt");
+        listSize = parser.readFileContents(FILE_NAME, tasks);
+        printWithLineBreak("Stored list size: " + listSize);
+        Scanner reader = new Scanner(System.in);
+        String userInput = reader.nextLine();
+
+        // Echoes the input, unless input == bye
```

Comment:
Is this comment relevant to the program anymore? I suppose the chatbot does not simply echo the entered command anymore.